python-checks/src/test/resources/checks/unusedLocalVariable.py

```diff
@@ -47,7 +47,8 @@ def function_with_lambdas():
 #             ^
 
 def using_tuples():
-    x, y = (1, 2)
+    x, y = (1, 2) # Noncompliant {{Replace unused local variable "y" with "_".}}
```

Comment:
Note that the rule description also needs to be updated, see the "Exceptions" section https://sonarsource.github.io/rspec/#/rspec/S1481/python 

```
for _ in range(10):
    do_something()
username, login, password = auth # This now raises
do_something_else(username, login)
```

Reply:
done